### PR TITLE
DKPY2 fixes to getting started, running examples, and developer setup…

### PR DESCRIPTION
**Pull request by hamishwillee**
This basically is first draft of changes to getting started, along with all the other minor link fixes an fixes to docs that made sense following the changes and which fix any bugs that showed up in links and rendering during the build. 

The getting started section is a lot simpler and I removed a lot of dependencies. Worth checking. I also removed the specific "running an example" section and instead now point to the examples section for that.

This does need a basic review.

### CODE REVIEW

**Comment by tcr3dr** [comment]
Rebased to master.

File: docs/guide/getting_started.rst [hunk 1]
```diff
@@ -21,7 +20,7 @@ Setting up the vehicle/autopilot
 For information on how to set up a vehicle (real and simulated) see:
 
 * :ref:`supported-companion-computers` for links to tested hardware/software configurations for a number of onboard Linux computers. 
-* :ref:`sitle_setup` for links explaining how to set up a simulated vehicle for Copter, Plane, or Rover.
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: `sitle`++

File: docs/guide/getting_started.rst [hunk 2]
```diff
@@ -40,11 +39,10 @@ If you are using Ubuntu or Debian Linux you can get most of the *DroneKit* depen
 
 .. code:: bash
 
-    sudo apt-get install python-pip python-dev python-opencv python-serial python-pyparsing python-wxgtk2.8
+    sudo apt-get install python-pip python-dev
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: isn't this great?

File: docs/guide/getting_started.rst [hunk 3]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
+.. _get_started_install_dk_windows:
 
-Install DroneKit-Python and its remaining dependencies (including `MAVProxy <http://tridge.github.io/MAVProxy/>`_) from the public PyPi repository:
+Installing DroneKit on Windows
+------------------------------
 
-.. code:: bash
+Set up a command line DroneKit-Python installation using *WinPython* (this Python distribution already includes most of the needed dependencies).
 
+<<<<<<< HEAD
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: I think you left this here from a git conflict, can you remove them please?

File: docs/guide/getting_started.rst [hunk 4]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
     sudo pip install pyparsing
     sudo pip install dronekit
+=======
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 5]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
+#. Download and run the correct `WinPython installer <http://sourceforge.net/projects/winpython/files/WinPython_2.7/2.7.6.4/>`_ (**v2.7**) for your platform (win32 vs win64).
+   
+   * Run the installer as an administrator (**Right-click** on file, select **Run as Administrator**). 
+   * When prompted for the destination location, specify **C:\\Program Files (x86)** 
+     (the default location is under the **Downloads** folder).
+>>>>>>> DKPY2 fixes to getting started, running examples, and developer setup windows. Consequent fixups to links elsewhere
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 6]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
-*DroneKit* is implemented as a *MAVProxy* module (MAVProxy is installed automatically with DroneKit). 
-The best way to load the *DroneKit* module into *MAVProxy* is to 
-`add it to the startup script <http://tridge.github.io/MAVProxy/mavinit.html>`_ (**mavinit.scr**).
-
-Linux/MAC OSX:
-
-.. code:: bash
+.. tip::
 
+<<<<<<< HEAD
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: git conflict

File: docs/guide/getting_started.rst [hunk 7]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
     echo "module load dronekit.module.api" >> ~/.mavinit.scr
+=======
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 8]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
+    The strings above are the same as you would use if connecting with MAVProxy. For other options see the 
+    `MAVProxy documentation <http://dronecode.github.io/MAVProxy/html/getting_started/starting.html#master>`_.
+>>>>>>> DKPY2 fixes to getting started, running examples, and developer setup windows. Consequent fixups to links elsewhere
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 9]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
 
-Windows:
+    
+You can start this simple script in the same way you would start any other standalone Python script. 
 
-.. code:: bash
+.. code-block:: bash
 
+<<<<<<< HEAD
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: git conflict

File: docs/guide/getting_started.rst [hunk 10]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
     echo module load dronekit.module.api >> %HOMEPATH%\AppData\Local\MAVProxy\mavinit.scr
+=======
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 11]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
+    python your_dronekit_script.py
+>>>>>>> DKPY2 fixes to getting started, running examples, and developer setup windows. Consequent fixups to links elsewhere
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 12]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
 
 
-Alternatively you can choose to manually (re)load *DroneKit* into *MAVProxy* every time you need it:
 
+<<<<<<< HEAD
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: moaaar conflicts!

File: docs/guide/getting_started.rst [hunk 13]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
 .. code-block:: bash
    :emphasize-lines: 1
 
     MANUAL> module load dronekit.module.api
     DroneAPI loaded
     MANUAL>
+=======
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 14]
```diff
@@ -61,151 +59,129 @@ installed when you get DroneKit-Python from the public PyPi repository:
...
+.. todo:: Connect method here needs to link to the function, but it isn't exported yet. Fix that once the API tidied.
+    
+>>>>>>> DKPY2 fixes to getting started, running examples, and developer setup windows. Consequent fixups to links elsewhere
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 15]
```diff
@@ -214,6 +190,7 @@ Alternatively you can choose to manually (re)load *DroneKit* into *MAVProxy* eve
 Running an app/example
 ======================
 
+<<<<<<< HEAD
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: I'm guessing this isn't fun right?

File: docs/guide/getting_started.rst [hunk 16]
```diff
@@ -289,5 +266,9 @@ The steps are:
        Got MAVLink msg: COMMAND_ACK {command : 11, result : 0}
        ...
 
+=======
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

File: docs/guide/getting_started.rst [hunk 17]
```diff
@@ -289,5 +266,9 @@ The steps are:
...
+This SDK has :ref:`numerous examples <example-toc>`. We recommend you start with :ref:`example-vehicle-state`, 
+which reads and writes :ref:`vehicle state and parameter <vehicle-information>` information. 
+>>>>>>> DKPY2 fixes to getting started, running examples, and developer setup windows. Consequent fixups to links elsewhere
```
Comment (mrpollo) [inline docs/guide/getting_started.rst]: same

**Comment by mrpollo** [comment]
looks good but has some git conflicts that need to be resolved, possibly squash this commit afterwards too

**Comment by tcr3dr** [comment]
Yeah the conflicts were probably me :) the only diff should be <<<OLD STUFF WITH "droneapi" replaced with "dronekit" ===== HAMISH'S CHANGES >>>>

**Comment by mrpollo** [comment]
to be fair sphinx makes it really hard to understand what's going on

**Comment by hamishwillee** [comment]
No guilty (me or sphinx). I'll try fix this up on Monday.

**Comment by hamishwillee** [comment]
@tcr3dr @mrpollo I removed that rubbish. Admitting guilt <slap on wrist> :-(  I made sure that this also refers to dronekit rather than droneapi, and removed the numpy dependency.

Ready to merge (though I will hopefully add further updates to examples, depending on how testing of other fixes goes).

File: docs/guide/getting_started.rst (hunk 2 above)
Comment (tcr3dr) [inline docs/guide/getting_started.rst]: > Pray to the God it fall down
> Pray to the cash it fall down

**Pull request merged**